src/app/screens/signPsbtRequest/index.tsx

```diff
@@ -137,6 +179,7 @@ function SignPsbtRequest() {
   const onSignPsbtConfirmed = async () => {
     try {
       if (isLedgerAccount(selectedAccount)) {
+        setIsModalVisible(true);
```

Comment:
if we include psbt signing in this pr we need to have the ux of the implementation reviewed by @QUDBS can you please record a small video to present

Reply:
Okay, first of all, here's BIP-322 signing video:
https://github.com/secretkeylabs/xverse-web-extension/assets/36603049/dd7d4143-c69d-4386-b2d2-9e508ec6b0c5

Reply:
And here's the PSBT signing video:
https://github.com/secretkeylabs/xverse-web-extension/assets/36603049/237c8d6d-0214-43da-b21d-baac8aae19ea

Reply:
@QUDBS @yknl for your attention please